app/components/bali/table/component.rb

```diff
@@ -31,6 +31,13 @@ def id
       end
 
       class MissingFilterForm < StandardError; end
+
+      private
+
+      def empty_table_row_id
+        empty_row_id = 'empty-table-row'
+        id ? "#{id}-#{empty_row_id}" : empty_row_id
```

Comment:
Creo que se puede simplificar de esta manera.

```suggestion
        [id, 'empty-table-row'].compact.join('-')
```